infer/src/base/Config.ml

```diff
@@ -2857,6 +2857,12 @@ and subtype_multirange =
     "Use the multirange subtyping domain. Used in the Java frontend and in biabduction."
 
 
+and suffix_match_changed_file =
+  CLOpt.mk_bool ~long:"postfix-match-changed-file"
+  ~default:false
+    "Match changed file name by suffix part of the file name."
```

Comment:
"changed file name" should be plural as there can be several. Also this could be clearer, eg
`"When computing the set of files to analyze using $(b,--changed-files-index), a file will be analyzed if a name in the changed files index is a suffix of its name."`

Reply:
Done